kmir/src/kmir/__main__.py

```diff
@@ -50,7 +50,7 @@ def exec_run(
 ) -> None:
     krun_output = KRunOutput[output.upper()]
     br = BugReport(Path(input_file).with_suffix('.bug_report.tar')) if bug_report else None
-    kmir = KMIR(definition_dir, definition_dir, bug_report=br)
+    kmir = KMIR(definition_dir, None, bug_report=br)
```

Comment:
Because I am passing `None`, both backends do need to be built. Maybe this is not preferable.